src/Pages/CandidatesTable.tsx

```diff
@@ -0,0 +1,47 @@
+import { atom, useAtom } from "jotai";
+import { Link, useParams } from "react-router-dom";
+import { candidateCount, filterData } from "../Components/CandidateTable/candidateTableAtoms";
+import { rank } from "../Components/CandidateTable/candidateTableTypes";
+import { companyName } from "../Router";
+import TestCandidateTable from "../TestCandidateTable";
+import Updater from "./Updater";
+
+export const filterToLinkData = (filterData: rank[], count: number) => (
```

Comment:
I know it should be done with `atomWithHash`, but still, it's really cool that you implemented this yourself. Good job!